tests/benchmarks/benchmark_runner.py

```diff
@@ -55,14 +66,11 @@ async def test_benchmark(retries):
 
         for prompt in benchmark.prompts:
             print("  Prompt:", prompt)
+            results[title][prompt] = {
+                "Success": 0,
+            }
             for i in range(1, retries + 1):
-                client = PythonClient(
-                    config=Config(
-                        auto_context=True,
-                        maximum_context=8000,
-                    )
-                )
-
+                client = PythonClient(cwd=codebase, config=benchmark.config)
```

Comment:
Previously I didn't need to specify the cwd but now I do. Not sure exactly why because I change the current working directory before each benchmark and PythonClient should use the current working directory if an argument isn't given.